faraday/aprs.py

```diff
@@ -693,14 +714,17 @@ def connectAPRSIS():
                 aprssock.sendall(logon_string)
 
             except IOError as e:
+                # Close socket and setup for next connection attempt
                 logger.error(e)
+                aprssock.close()
+                aprssock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 
             else:
                 logger.info("Connection successful!")
                 return aprssock
                 break
```

Comment:
Not this change, but `break` after `return` does nothing.

Reply:
Removed